build/deps/tigris_fdb.Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM foundationdb/foundationdb:6.3.23
+
+# FROM tigrisdata/foundationdb:7.1.7
```

Comment:
This and `tigris_typesense` would be uncommented in the next PR, I guess?

Reply:
When we upgrade to the next Tigris alpha. Currently, I could not make it work